lib/serve/rest-api/src/utils/generate_litellm_config.py

```diff
@@ -51,6 +52,22 @@ def generate_config(filepath: str) -> None:
     config_models = config_contents["model_list"] or []  # ensure config_models is a list and not None
     config_models.extend(litellm_model_params)
     config_contents["model_list"] = config_models
+
+    # Get database connection info
+    db_param_response = ssm_client.get_parameter(Name=os.environ["LITELLM_DB_INFO_PS_NAME"])
+    db_params = json.loads(db_param_response["Parameter"]["Value"])
+    secrets_response = secrets_client.get_secret_value(SecretId=db_params["passwordSecretId"])
+    password = json.loads(secrets_response["SecretString"])["password"]
+    connection_str = f"postgresql://{db_params['username']}:{password}@{db_params['dbHost']}:5432/{db_params['dbName']}"
```

Comment:
I know this is pretty standard but should we make the port configurable too?

Reply:
Yea I can do that- I'll update the schema to add it there by default- since I'm using the same rds config as the RAG repo- I'll need to update there too.